http/check_service.go

```diff
@@ -47,6 +49,7 @@ type CheckHandler struct {
 	influxdb.HTTPErrorHandler
 	Logger *zap.Logger
 
+	TaskService                influxdb.TaskService
```

Comment:
i think we should do this in KV. instead of the http layer. Just make check.Status inherit from task. in the findCheck service. There is no need to any extra hack for creation and update.

Reply:
If we push this down into the KV layer then we will probably want to do a lookup from the store at the end of create, update and patch. Does anyone have objections to that?

Reply:
As opposed to modifying the check in place.